object-construction-checker/build.gradle

```diff
@@ -57,40 +55,6 @@ tasks.withType(JavaCompile).all {
     options.compilerArgs.add("-Xlint:all")
 }
 
-def tsbcVersion = '0.1-SNAPSHOT'
-
-task qualJar(type: Jar) {
-    baseName 'object-construction-qual'
-    from (sourceSets.main.output) {
-        include "**/qual/**"
-    }
-}
-
-publishing {
-    publications {
-        checker(MavenPublication) {
-            groupId = 'org.checkerframework'
-            artifactId = 'object-construction'
-            version = "${tsbcVersion}"
-
-            from components.java
-        }
-
-        qual(MavenPublication) {
-            groupId = 'org.checkerframework'
-            artifactId = 'object-construction-qual'
-            version = "${tsbcVersion}"
-            artifact qualJar
-            pom.withXml {
-                def depNode = asNode().appendNode("dependencies").appendNode("dependency")
-                depNode.appendNode("groupId", "org.checkerframework")
-                depNode.appendNode("artifactId", "returnsrcvr-qual")
-                depNode.appendNode("version", "0.1-SNAPSHOT")
-            }
```

Comment:
The code directly above is what currently makes the qual package depend directly on `returnsrcvr-qual`